Python/import.c

```diff
@@ -3721,6 +3721,10 @@ import_find_and_load(PyThreadState *tstate, PyObject *abs_name)
 
         accumulated = accumulated_copy + cum;
     }
+    if (_PySys_Audit(tstate, "import.complete", "OO", abs_name,
```

Comment:
```suggestion
    if (_PySys_Audit(tstate, "import/module", "OO", abs_name,
```

We have a few other pairs of events and we separate the kind of the result from the name with a slash.

Reply:
I think that your renaming makes sense if you view this as primarily a means of auditing the loaded module, but `import/complete` would make clear that it occurs at the end of the load process (which is my use-case - allowing programmatic import profiling). What is the reason that you feel that `import/module` is a better name?

Reply:
> What is the reason that you feel that `import/module` is a better name?

Because the existing events are `ctypes.dlsym/handle`, `ctypes.cdata/buffer`, `builtins.input/result`, `sqlite3.connect/handle`, and `winreg.OpenKey/result`. All but the last one specify the _kind_ of the result (`input` being incredibly generic, and `winreg.OpenKey/hkey` would've been a better name) - note _kind_, not _type_.

So `import/module` is specifying the kind of the result. I'd settle for `import/result`, but I think `import/module` is better (unless it also gets raised on errors, but that's going to be a pain to code up and probably not worth it).

Reply:
Done. Good point.